Marlin/src/lcd/extui/anycubic_vyper/dgus_tft.cpp

```diff
@@ -98,7 +98,7 @@ namespace Anycubic {
   bool DgusTFT::data_received;
   uint8_t DgusTFT::data_buf[DATA_BUF_SIZE];
   uint8_t DgusTFT::data_index;
-  uint32_t DgusTFT::page_index_now, DgusTFT::page_index_last, DgusTFT::page_index_last_2;
+  uint16_t DgusTFT::page_index_now, DgusTFT::page_index_last, DgusTFT::page_index_last_2;
```

Comment:
@Bob-the-Kuhn — This commit formalizes the face that the values being provided are being sent with two byte transfers. … Assuming that the endian changes from the previous commit are ok. I would expect the changes related to endianness are the most likely culprit for any new problems.